web/src/locales/en.json

```
@@ -95,6 +95,8 @@
 	"signin.password.placeholder": "Enter your password",
 	"signin.button": "Sign in",
 	"signin.error.incorrectCredentials": "Invalid credentials. Either the username or password are incorrect.",
+	"signOut": "Sign out",
```

Should we keep consistency with sign in?
```suggestion
	"signOut.button": "Sign out",
```